index.html

```diff
@@ -84,6 +84,263 @@ <h1 class="primary-headline-text">Hey there. I’m Ansderly. <br>
         </div>
     </section>
 
+    <section class="recent-work-container">
+        <div class="recent-work-title-container">
+            <div class="recent-work-title">
+                <h1 class="text-recent-work">My Recent Works</h1>
+                <div class="line-work-container">
+                    <div class="line-work"></div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="single-card-container">
+            <div class="single-card">
+                <div class="img-single-work">
+                </div>
+                <div class="infos-container">
+                    <div class="name-single-work">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                    </div>
+                    <div class="stack-single-work-container">
+                        <ul class="stack-single-container">
+                            <li class="stack"><a href="#">Ruby on rail</a></li>
+                            <li class="stack"><a href="#">css</a></li>
+                            <li class="stack"><a href="#">Javascript</a></li>
+                            <li class="stack"><a href="#">html</a></li>
+                        </ul>
+                    </div>
+                    <div class="see-project-container">
+                        <div class="see-project-button">
+                            <a class="see-project-text" href="#">See Project</a>
```

Comment:
- [Optional]  Please add a hover and pressed state to all your buttons as indicated in the figma template.
![btn3](https://user-images.githubusercontent.com/94551484/205368401-27bba95f-aed2-44cd-a105-870d5246b99f.png)